.all-contributorsrc

```diff
@@ -2649,7 +2649,8 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/37468543?v=4",
       "profile": "https://github.com/meraldoantonio",
       "contributions": [
-        "doc"
+        "doc",
+        "bug"
```

Comment:
`"bug"` is for diagnosing or reporting bugs, but not bugfixes. Bugfixes fall under the `"code"` badge, so you can add both.